### ISSUE TITLE
The Keyboard navigation doesn't work properly on dropdown menu

### ISSUE THREAD

**Issue by Anenth** [react-bootstrap/react-bootstrap#2532]
The keyboard navigation(up/down arrow) doesn't work properly. 
When down arrow is pressed it goes to the 3rd element than the 2nd element.

![](http://g.recordit.co/J2n9sTnZot.gif)

**Comment by Anenth** [comment]
Fixed at #2533

**Comment by taion** [comment]
Closing per https://github.com/react-bootstrap/react-bootstrap/pull/2533#issuecomment-325452382 given inability to reproduce.

**Comment by etedesco** [comment]
I have this issue.
You can reproduce the issue, using this link
http://www.gulamshakir.com/apps/s3calc/

Problem found using "react-bootstrap": "0.32.1" 
Anybody can help me ?

**Comment by spentacular** [comment]
I'm having the same problem. Our current dropdown structure looks like this:

```
<DropdownButton
  id="dropdown-id"
>
  <MenuItem>Menu Item 1</MenuItem>
  <MenuItem>Menu Item 2</MenuItem>
  <MenuItem>Menu Item 3</MenuItem>
  <MenuItem divider={true} />
  <MenuItem>Menu Item 4</MenuItem>
  <MenuItem>Menu Item 5</MenuItem>
  <MenuItem>Menu Item 6</MenuItem>
  <MenuItem>Menu Item 7</MenuItem>
  <MenuItem>Menu Item 8</MenuItem>
  <MenuItem>Menu Item 9</MenuItem>
</DropdownButton>
```

I'm on `0.31.3`, but have tried upgrading to `0.32.1` and I'm getting the same thing.

The menu items in the DOM all have a `tabindex="-1"`

Ours functions the same way as @etedesco and it's only arrowing on the odd ones, not the even ones.

I've tried using the custom dropdown format and putting everything in either a `Dropdown.Toggle` or `Dropdown.Menu`, but I'm getting the same effect.

**Comment by spentacular** [comment]
I'm seeing that the next menu item is being moved to _then_ this is triggered:

https://github.com/react-bootstrap/react-bootstrap/blob/master/src/DropdownMenu.js#L48

So the active element is moving +1 before any of the internal next/previous has been calculated.

**Comment by etedesco** [comment]
I removed bootstrap.js import and it works !!!  
Thanks a lot @spentacular

**Comment by 416serg** [comment]
@etedesco where did you remove `bootstrap.js` from? I am having the same issue right now and it looks like your [example](http://www.gulamshakir.com/apps/s3calc/?) is still jumping over 1 `MenuItem`

**Comment by etedesco** [comment]
I removed this import in my index.js file
import 'bootstrap/dist/js/bootstrap.min.js';

**Comment by DavidFreetly** [comment]
I have a situation where my react components are just a small part of a larger html page that requires jquery and bootstrap so my fix for this issue was to move my react bundle above the others in my html file.

<script src="./_files/js/main.f7db35db.js" type="text/javascript"></script>
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js" type="text/javascript"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js" type="text/javascript"></script>
